src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margins and padding using * selector. It applies to all the elements, which is not good for performance. 
Better specify the elements you want to reset